### DISCUSSION TITLE
Improvements to the Name Pages

### DISCUSSION THREAD

**Discussion by mo-nathan** [MushroomObserver/mushroom-observer#1903] [General]
The current name pages are much improved, but there are still a few features I'd like to see.  First, I think we should have a way to select specific image thumbnails for names.  This would allow pages like: https://mushroomobserver.org/names?pattern=Amanita to have some characteristic images of those names that we have images for.  It would also help with the name pages for projects (e.g., https://mushroomobserver.org/checklist?project_id=336) and probably several other places where names appear without any image content.

I think there are two many options under "Observations of:".  To me "this taxon, any name" and "any taxon, this name proposed" should probably go since these are just unions of some of the other options.  I also wonder if there should be an "Images of:" option especially since the "Show Images" link doesn't currently work correctly.

**Comment by JoeCohen** [comment]
I agree with simplifying the options for Observations of. 
Maybe we need some more user input on this. I don't now which -- if any of the options -- are used.
For me, the two most useful functions (regardless what label we put in the link) are:
- Observations of this thing (taxon) regardless which Name was applied to it. I.e., I want to see Observations of blewits, regardless whether the consensus is _Clitocybe nuda_ or _Lepista nuda_ (or any other synonym). "this taxon, any name" should do this.
- Observations of taxa with which this taxon is confused. "other taxa, this taxon proposed" should do this.

 I suspect (most?) other cases can be handled via pattern search.
 But probably so can the above two.  @nimmolo: Would doing them all by a link to a pattern search make this page less expensive?

**Reply by nimmolo** [reply]
Something to consider: These links _were_ originally expensive (due to the _counts_), but after a couple full days of experimentation, I figured out a way to agglomerate them into a single mother query, and get the counts by subsetting that query (using operations on the original ruby object, not separate AR queries). So i'm no longer feeling the same urgency to purge. 

I'm feeling the urgency to protect my baby! Hahaha, just kidding. Winnow them down if it makes sense. But just know that doing the similar counts is no longer costly.

**Reply by JoeCohen** [reply]
Thanks @nimmolo! I overlooked the counts, which are useful.

**Comment by mo-nathan** [comment]
Yeah, the getting rid of links things isn't urgent.  I'm fine with what @JoeCohen wants.
